sos/report/plugins/apt.py

```diff
@@ -32,7 +32,8 @@ def setup(self):
             "apt-get check",
             "apt-config dump",
             "apt-cache stats",
-            "apt-cache policy"
+            "apt-cache policy",
+            "apt-mark showhold"
```

Comment:
minor nit, can you add a comma at the end, that way it's one-line change the next time.